project.mml

```diff
@@ -1411,6 +1411,22 @@ Layer:
     properties:
       cache-features: true
       minzoom: 12
+  - id: stop-positions
+    geometry: point
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+          way, 
+          tags->'public_transport' as public_transport,
```

Comment:
With only one possible value, there's no reason to include this in the output.

Reply:
Fixed